Xamarin.Forms.Core/FlexEnums.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+namespace Xamarin.Forms
+{
+	//TODO require a typeConverter for css values
+	public enum FlexJustify
```

Comment:
As Xamarin.Flex enums sometimes have values not supported for some properties, we redefine them